src/components/OrderDetails.vue

```diff
@@ -24,13 +24,13 @@
             <p class="text-lg font-semibold text-gray-800">{{ formatCurrency(props.orderDetails.down_payment) }}</p>
         </div>
         <p class="mt-4 flex text-lg font-semibold text-gray-800">
-            {{ formatCurrency(props.orderDetails.repayment) }}<span class="text-sm font-semibold leading-8 tracking-normal text-gray-500">/monthly</span>
+            {{ formatCurrency(props.orderDetails.amortizations[0].expected_amount) }}<span class="text-sm font-semibold leading-8 tracking-normal text-gray-500">/ {{ findRepayment(props.orderDetails.repayment_cycle_id, repayment_cycle ) }}</span>
```

Comment:
Can we use optional chaining? To be save
`props.orderDetails?.amortizations?.[0]?.expected_amount`